docs/source/features/mixed_precision.rst

```diff
@@ -13,8 +13,49 @@ NeMo Framework supports half-precision FP16 and BF16 computation training via Me
 This training recipe uses half-precision in all layer computation keeping the model states (optimizer states and master parameters) in single-precision.
 To avoid repeated data type casting at each layer computation, Megatron Core keeps a separate copy of half-precision parameters that is updated after each optimizer step.
 
-Half-precision training is enabled when setting ``precision`` to either of ``fp16-mixed`` or ``bf16-mixed`` along with  ``megatron_amp_O2=true``.
```

Comment:
Line 9/9 fix capitalization in heading above

Half-Precision Training
===============